### DISCUSSION TITLE
num_search_workers and log_search_progress

### DISCUSSION THREAD

**Discussion by rahulj2409** [google/or-tools#2662] [Routing (and legacy CP) questions] [answered]
Python
I am currently following the sample code for the Vehicle Routing Problem given by https://developers.google.com/optimization/routing/vrptw. I wanted to change the "num_search_workers" variable/see what the current value is set to. However, I don't know where that value is located/how to change it. Is it search_parameters.num_search_workers (that didn't work). I tried routing.num_search_workers, that didn't work, I tried routing.Solver().num_search_workers too. How do we change those values

Thanks!

**Comment by ghost** [comment, marked as answer]
That parameters is for the CP-SAT solver and is not related to routing

**Reply by rahulj2409** [reply]
Is there a similiar thing used for the routing model? IE is there any parallel-optimization core stuff that can be used when solving for the Vehicle Routing Problem

**Reply by lperron** [reply]
Not currently.
